dds_web/api/project.py

```diff
@@ -949,3 +949,28 @@ def put(self):
             "ok": True,
             "message": f"Project {project_id} was set to {'busy' if set_to_busy else 'not busy'}.",
         }
+
+
+class ProjectInfo(flask_restful.Resource):
+    """Get information for a specific project."""
+
+    @auth.login_required
+    @logging_bind_request
+    @handle_validation_errors
```

Comment:
Add decorators to handle databaseerrors
```suggestion
    @handle_db_error
```